pulp_ansible/app/tasks/collections.py

```diff
@@ -195,6 +196,17 @@ def create_collection_from_importer(importer_result):
     return collection_version
 
 
+def add_collection_to_repository(repository_pk, collection_version_pk):
```

Comment:
This is presumably close enough to the same as https://github.com/ansible/galaxy_ng/blob/master/galaxy_ng/app/tasks/promotion.py#L4
that galaxy_ng should be able to use pulpcore.app.tasks.collections.add_collection_to_repository() once it is merged?

Reply:
pinging @awcrosby

Reply:
Yeah that is something we could look at replacing with this